Dockerfile

```diff
@@ -2,13 +2,23 @@ FROM centos:7
 MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
 
 RUN yum update -y && \
-    yum install -y golang openscap-scanner && \
+    yum install -y golang openscap-scanner git && \
     yum clean all
 
-WORKDIR /var/lib/image-inspector
+RUN GOBIN=/usr/bin \
+    GOPATH=/go \
+    go get github.com/rancher/trash
 
 COPY .  /go/src/github.com/openshift/image-inspector
 
+WORKDIR /go/src/github.com/openshift/image-inspector/
+
+RUN GOBIN=/usr/bin \
+    GOPATH=/go \
+    trash
```

Comment:
@simon3z come to think of it, we don't actually need this here since we are committing the `vendor` folder. As long as we make sure to keep `vendor` and `vendor.conf` updated in each commit, this line is not needed.